{"level": "info", "log_string": "Successfully called https://reqres.in/api/users?page=2", "timestamp": "2024-05-14T12:07:50.986006Z", "metadata": {"source": "api6.log"}}
{"level": "success", "log_string": "Response: {'page': 2, 'per_page': 6, 'total': 12, 'total_pages': 2, 'data': [{'id': 7, 'email': 'michael.lawson@reqres.in', 'first_name': 'Michael', 'last_name': 'Lawson', 'avatar': 'https://reqres.in/img/faces/7-image.jpg'}, {'id': 8, 'email': 'lindsay.ferguson@reqres.in', 'first_name': 'Lindsay', 'last_name': 'Ferguson', 'avatar': 'https://reqres.in/img/faces/8-image.jpg'}, {'id': 9, 'email': 'tobias.funke@reqres.in', 'first_name': 'Tobias', 'last_name': 'Funke', 'avatar': 'https://reqres.in/img/faces/9-image.jpg'}, {'id': 10, 'email': 'byron.fields@reqres.in', 'first_name': 'Byron', 'last_name': 'Fields', 'avatar': 'https://reqres.in/img/faces/10-image.jpg'}, {'id': 11, 'email': 'george.edwards@reqres.in', 'first_name': 'George', 'last_name': 'Edwards', 'avatar': 'https://reqres.in/img/faces/11-image.jpg'}, {'id': 12, 'email': 'rachel.howell@reqres.in', 'first_name': 'Rachel', 'last_name': 'Howell', 'avatar': 'https://reqres.in/img/faces/12-image.jpg'}], 'support': {'url': 'https://reqres.in/#support-heading', 'text': 'To keep ReqRes free, contributions towards server costs are appreciated!'}}", "timestamp": "2024-05-14T12:07:50.987002Z", "metadata": {"source": "api6.log"}}
{"level": "info", "log_string": "Successfully called https://api.ipify.org?format=json", "timestamp": "2024-05-14T12:09:56.307203Z", "metadata": {"source": "api6.log"}}
{"level": "success", "log_string": "Response: {'ip': '106.213.34.64'}", "timestamp": "2024-05-14T12:09:56.307203Z", "metadata": {"source": "api6.log"}}
{"level": "error", "log_string": "Other error occurred: HTTPSConnectionPool(host='api.ipify.org', port=443): Max retries exceeded with url: /?format=json (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x00000192CF787050>: Failed to resolve 'api.ipify.org' ([Errno 11001] getaddrinfo failed)\"))", "timestamp": "2024-05-14T12:10:16.610126Z", "metadata": {"source": "api6.log"}}
{"level": "error", "log_string": "Other error occurred: HTTPSConnectionPool(host='api.ipify.org', port=443): Max retries exceeded with url: /?format=json (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x0000022775787050>: Failed to resolve 'api.ipify.org' ([Errno 11001] getaddrinfo failed)\"))", "timestamp": "2024-05-14T12:31:36.007736Z", "metadata": {"source": "api6.log"}}
